templates/katello.yaml.erb

```diff
@@ -18,14 +18,15 @@
     :oauth_secret: <%= scope['katello_devel::oauth_secret'] %>
     :ca_cert_file: <%= scope['katello_devel::candlepin_ca_cert'] %>
 
+  :candlepin_events:
+    :ssl_cert_file: <%= scope['certs::candlepin::client_cert'] %>
```

Comment:
Just thinking out loud: this feels wrong. The class `certs::candlepin` should be executed on the candlepin server. A katello server doesn't have to be the same, so this class may not be included. In the current devel module it's not that relevant, but for `puppet-katello` it's incorrect.